src/components/Common/SearchInput.tsx

```diff
@@ -336,22 +355,21 @@ const SearchByMultipleFields: React.FC<SearchByMultipleFieldsProps> = ({
             </PopoverContent>
           </Popover>
         )}
-        <div className="w-full">{renderSearchInput}</div>
+        <div className="w-full">{renderInput}</div>
```

Comment:
why not just inline that component here? 🤔 

<img width="439" alt="image" src="https://github.com/user-attachments/assets/53ee2b8b-0eba-437f-9f37-beb8059ae9e0" />